.github/workflows/ci.yml

```diff
@@ -507,6 +507,85 @@ jobs:
           path: asv_bench/benchmarks.log
         if: failure()
 
+  # This should be migrated to push-to-master.yml before merging!!!
+  test-all-unidist:
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
  test-all-unidist:
    needs: [lint-flake8, lint-black, lint-mypy, test-api, test-headers]
    runs-on: ubuntu-latest
```

Reply:
If we agree to keep the job in this file, I will address this.

Reply:
fixed